packages/arb-token-bridge-ui/src/util/__tests__/networks.test.ts

```diff
@@ -0,0 +1,130 @@
+import { addCustomChain } from '@arbitrum/sdk/dist/lib/dataEntities/networks'
+
+import { ChainId, getBaseChainIdByChainId, xaiTestnet } from '../networks'
+
+beforeAll(() => {
+  // add local
+  addCustomChain({
```

Comment:
We don't seem to be using these chains anywhere in the tests, better to remove it then?

Reply:
we do, that's why it's added

```
      expect(
        getBaseChainIdByChainId({
          chainId: ChainId.Local
        })
      ).toBe(ChainId.Local)
```

Reply:
i'll add a test for arbitrum local